data/json/items/melee.json

```diff
@@ -2283,5 +2283,43 @@
     "techniques": "RAPID",
     "qualities": [ [ "CUT", 1 ], [ "CUT_FINE", 1 ], [ "BUTCHER", 7 ] ],
     "flags": [ "STAB", "SHEATH_KNIFE" ]
+  },
+  {
+    "id": "2h_flail_steel",
+    "type": "GENERIC",
+	"category": "weapons",
+    "name": "war flail",
+    "name_plural": "war flails",
```

Comment:
Redundant.

Reply:
Can you please clarify?, I genuinely do not understand what is redundant there, I would love to fix it but again I don't understand where the redundancy is.

Reply:
Clarifying: game engine automatically adds trailing 's' to the text in `name` field if no `name_plural` field is defined. In your example `war flails` = `war flail` + automatically added trailing `s`, so `name_plural` field in this case is redundant.

Reply:
Oh!, I didn't know that, fixing it.